ansible/files/steps/zeppelin/zeppelin.sh

```diff
@@ -17,6 +18,17 @@ nohup Xvfb &
 
 echo 'export DISPLAY=:0.0' | sudo tee -a /etc/zeppelin/conf/zeppelin-env.sh
 
+# Preload Scala packages
+repositories="https://oss.sonatype.org/content/repositories/snapshots"
```

Comment:
Is this considered a trusted source? Do you think we need to mirror this in anyway for reliability/performance?